src/components/LoggedCabinet/LoggedCabinet.js

```diff
@@ -9,7 +9,8 @@ import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 
 import { useStyles } from './LoggedCabinet.styles';
 import { setThemeMode } from '../../redux/theme/theme.actions';
-import { LANGUAGE, CABINET_OPTIONS_LOGGED } from '../../configs';
+import { LANGUAGE } from '../../configs';
```

Comment:
please take language not from config but from store

Reply:
Resolved

Reply:
Resolved